analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -82,28 +90,14 @@ public void Disposed_Using_WithExpressions()
     public void Disposed_Using_Parameters(IDisposable param1)
     {
         param1.Dispose();
-        param1.Dispose(); // FIXME Non-compliant
-    }
-
-    public void Close_ParametersOfDifferentTypes(IInterface1 interface1, IDisposable interface2)
```

Comment:
I deleted these tests as I don't see the point of having them here.

Reply:
It was ensuring that disposing instance A does not also consider instance B disposed in the same method.

I'd keep it